entrypoint/selinux/selinux_test.go

```diff
@@ -47,9 +47,9 @@ var _ = Describe("Test:", func() {
 		testcase.TestSelinuxContext(cluster)
 	})
 
-	if customflag.ServiceFlag.InstallMode.String() != "" {
+	if customflag.ServiceFlag.UpgradeMode.String() != "" {
```

Comment:
Can we default `UpgradeMode` to `InstallMode` in case `UpgradeMode` is empty? I guess we are using `UpgradeMode` to allow install via commit?

Reply:
Just noticed that `UpgradeMode` is a struct... so need to see how it is implemented

Reply:
removing